README.md

```diff
@@ -78,6 +78,11 @@ see [Migrate from the SignalFx Java Agent](https://docs.splunk.com/Observability
 
 <!-- Comments, spacing, empty and new lines in the section below are intentional, please do not modify them! -->
 <!--DEV_DOCS_WARNING-->
+<!--DEV_DOCS_WARNING_START-->
+The documentation below refers to the in development version of this package. Docs for the latest version ([v1.5.0](https://github.com/signalfx/splunk-otel-java/releases/tag/v1.5.0)) can be found [here](https://github.com/signalfx/splunk-otel-java/blob/v1.5.0/README.md).
```

Comment:
Probably only a very minor help here since the other 2 places still need updating each version, but you can use the redirector url for the latest release.

```suggestion
The documentation below refers to the in development version of this package. Docs for the latest version ([v1.5.0](https://github.com/signalfx/splunk-otel-java/releases/latest)) can be found [here](https://github.com/signalfx/splunk-otel-java/blob/v1.5.0/README.md).
```

Reply:
Sure, why not -- updated the `post-release.sh` script too.